docs/source/endpoints/relations.md

```diff
@@ -0,0 +1,246 @@
+---
+myst:
+  html_meta:
+    'description': 'Create, query, and delete relations between content items with the /@relations endpoint.'
+    'property=og:description': 'Create, query, and delete relations between content items with the /@relations endpoint.'
+    'property=og:title': 'Relations'
+    'keywords': 'Plone, plone.restapi, REST, API, relations, service, endpoint'
+---
+
+(restapi-relations-label)=
+
+# Relations
+
+Plone's relations represent binary relationships between content objects.
+
+A single relation is defined by source, target and relation name.
+
+Relations are either defined by content type schema fields (`RelationChoice` or `RelationList`) or are of type `isReferencing` or `iterate-working-copy`.
+
+- Relations based on fields of a content type schema are editable by users.
+- Relations `isReferencing` (block text links to a Plone content object) and `iterate-working-copy` (working copy is enabled and the content object is a working copy) are not editable. They are created and deleted with links in text, respectively creating and deleting working copies.
+
+Relations can be created, queried, and deleted by interacting through the `@relations` endpoint on the site root.
+Querying relations with the `@relations` endpoint requires the `zope2.View` permission on the source object and target object.
+Therefore results include relations if and only if source and target are accessible by the querying user.
+Creating and deleting relations requires the respective permission on the content object.
```

Comment:
You have to have the View permission on the source and target in order to create or delete a relation? Or some other permission?

Reply:
Yes, the text is updated to:
Creating and deleting relations requires `zope2.View` permission on the target object and `cmf.ModifyPortalContent` permission on the source object.`